amit/html/Assigment2/ASSIn1.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+ <html>
+    <head>
+          <title>this is my first html</title>
+    </head>
+           <h1> <center> Google </center> </h1>
+        This article is about the company.
+           <h1> From the garbage to the Googleplex </h1>
+       <p> The Google story begins in 1995 at <b>Stanford university.</b><br> <b><i>Larry Page</i></b> was considering stanford for grad School and <b><i>Sergey Brin,</i></b> a student there,was assigned to Show him around.
+       </p>
+       <p> By Some accounts,they disagreed about nearly everything during that first meeting.but by the following year they struck a <mark> partnership.</mark><br>
+           Working from their dorm rooms,they built a search engine that used links to determine the importance of individual pages on the <font color="blue">World Wide Web.</font>
+       </p>
+           They called this search engine <b>Backrub</b>
+    <hr>
+    </hr>
+           <body style="background-color: pink;"></body>
+          <h1> <center> Google <center> </h1>
+               This article is about the company.
+          <h1> From the garbage to the Googleplex </h1>
+        <p> The Google story begins in 1995 at <b>Stanford university.</b><br> <b><i>Larry Page</i></b> was considering stanford for grad School and <b><i>Sergey Brin,</i></b> a student there,was assigned to Show him around.
+        </p>
+        <p> By Some accounts,they disagreed about nearly everything during that first meeting.but by the following year they struck a <mark> partnership.</mark><br>
+            Working from their dorm rooms,they built a search engine that used links to determine the importance of individual pages on the <font color="blue">World Wide Web.</font>
+        </p>They called this search engine <font color="black" style=background-color:blue>.Backrub</font>
+    <hr>
+    </hr>
+          <h2>Initial public offering </h2><br>
+         Google's Initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+         At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for30 20years,Until the year 2024.<br>
+           <center><font color="green" ><b> Thank You!!</b></font></center>
+    </html>
+ 
```

Comment:
remove extra lines which is present at the end of file only one line space expected after </html> tag